test/Interop/Cxx/operators/member-out-of-line.swift

```diff
@@ -0,0 +1,26 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-clang -c %S/Inputs/member-out-of-line.cpp -I %S/Inputs -o %t/member-out-of-line.o -std=c++17
```

Comment:
What's C++17 specific about this test?

Reply:
Nothing. I just copied the line. Not sure if we're targeting consistency.

Reply:
I was wrong. It's needed for the designated initializers.